10-inference.Rmd

```diff
@@ -0,0 +1,806 @@
+set.seed(2)
+samples_2 <- rep_sample_n(virtual_box, size = 40)
+choc_sample_2 <- summarize(samples_2, n = sum(flavour == "chocolate"),
+                                        prop = sum(flavour == "chocolate") / 40)
+choc_sample_2
+```
+
+Notice that we get a different value for our statistic this time. The
+proportion of chocolate Timbits in this sample is `r round(choc_sample_2$prop, 2)`. 
+If we were to do this again, another random sample could also give a
+different result. Statistics vary from sample to sample 
+due to **sampling variability**. 
+
+But just how much should we expect the statistics of our random
+samples to vary? In order to understand this, we will simulate more samples
+of size 40 from our population of Timbits, and calculate the 
+proportion of chocolate Timbits in each sample. We can then
+construct the distribution of sample proportions we calculate. The distribution
+of the statistic for all possible samples of size $n$ from a population is
+called a **sampling distribution**. The sampling distribution will help us see
+how much we would expect our sample proportions from this population to vary
+for samples of size 40. Below we again use the `rep_sample_n` to take samples
+of size 40 from our population of Timbits, but we set the `reps` argument
+to specify the number of samples to take.
+
+```{r 11-example-proportions5, echo = TRUE, message = FALSE, warning = FALSE}
+samples <- rep_sample_n(virtual_box, size = 40, reps = 1000)
+head(samples)
+tail(samples)
+```
+
+Notice the column `replicate` is indicating the replicate with which each
+Timbit belongs. Since we took 1000 samples of size 40, there are 1000 replicates.
+
+```{r 11-example-proportions6, echo = TRUE, message = FALSE, warning = FALSE}
+sample_estimates <- samples %>% 
+    group_by(replicate) %>% 
+    summarise(sample_proportion = sum(flavour == "chocolate") / 40)
+head(sample_estimates)
+tail(sample_estimates)
+```
+
+```{r 11-example-proportions7, echo = TRUE, message = FALSE, warning = FALSE,fig.cap = "Sampling distribution of the sample proportion for sample size 40"}
+sampling_distribution <-  ggplot(sample_estimates, aes(x = sample_proportion)) +
+    geom_histogram(fill="#0072B2", color="#e9ecef", binwidth = 0.05) +
+    xlab("Sample proportions") 
+sampling_distribution
+```
+
+The sampling distribution appears to be bell-shaped with one peak. It is
+centered around `r round(mean(sample_estimates$sample_proportion),1)` and the
+sample proportions range from about `r round(min(sample_estimates$sample_proportion), 1)` to 
+about `r round(max(sample_estimates$sample_proportion), 1)`. In fact, we can calculate
+the mean and standard deviation of the sample proportions. 
+
+```{r 11-example-proportions8, echo = TRUE, message = FALSE, warning = FALSE}
+sample_estimates %>% 
+  summarise(mean = mean(sample_proportion), sd = sd(sample_proportion))
+```
+
+We notice that the sample proportions are centred around the population
```

Comment:
We need more oomf here! It's actually pretty cool and incredible that the centre of the sampling distribution is the same as the population parameter. Here we don't actually see that, because we didn't take enough replicates. Let's actually take 10,000 or even 15,000 reps in this book. Our laptops are fast enough and then these two numbers will be closer, and our argument will be more convincing. 

Also - we need to bring this back to what we are doing! This distributions tells us what we might expect when taking a single random sample. And if we can somehow learn how to predict what sampling distributions should look like for any sample, we can use this knowledge to report more than just a single point estimate when we report our estimation results when we are in the more common, and realistic scenario of only having a single sample.

Reply:
upped the reps to 15000, added a summary paragraph at the end of proportions section